nicegui/server.py

```diff
@@ -27,10 +28,15 @@ def create_singleton(cls, config: CustomServerConfig) -> None:
     def run(self, sockets: Optional[List[socket.socket]] = None) -> None:
         self.instance = self
         assert isinstance(self.config, CustomServerConfig)
-        if self.config.method_queue is not None and self.config.response_queue is not None:
+        if (
+            self.config.method_queue is not None
+            and self.config.response_queue is not None
+            and self.config.drop_queue is not None
```

Comment:
```suggestion
            self.config.method_queue is not None and
            self.config.response_queue is not None and
            self.config.drop_queue is not None
```

I prefer this for its symmetry.